04 - Guides, Workflows, & Courses/Guides/Using Obsidian as a TTRPG Campaign Manager

```diff
@@ -0,0 +1,28 @@
+---
+aliases: 
+- Josh Plunkett
+- Obsidian TTRPG
```

Comment:
I'm not confident of these aliases, but should a specific page be given a much more general alias - will that cause confusion, perhaps if we have other pages about the more general Obsidian TTRPG topic?

Reply:
I've removed them. Just had my social media hat on where I tag everything to trigger the Google search algorithms.

Reply:
Thanks!